prometheus/resources.tf

```diff
@@ -14,7 +14,7 @@ resource "cloudfoundry_app" "prometheus" {
   docker_credentials = var.docker_credentials
   environment = {
     PROM_CONFIG          = local.config_file
-    ALERT_RULES          = var.alert_rules
+    ALERT_RULES          = var.alert_rules != "" ? local.config_file : local.app_alert_rules
```

Comment:
Can we append APP_ALERT_RULES to ALERT_RULES just like we append POSTGRES_ALERT_RULES?

Reply:
I tried that first but it's failing since `groups` is duplicated in alert_rules and local.app_alert_rules.  So we have to use one of those 2 alert_rules objects.